docs/zkapps/o1js/index.mdx

```diff
@@ -36,18 +36,31 @@ function knowsPreimage(preimage: Field) {
 }
 
 const expectedHash =
-  Field(0x1d444102d9e8da6d566467defcc446e8c1c3a3616d059facadbfd674afbc37ecn);
+  0x1d444102d9e8da6d566467defcc446e8c1c3a3616d059facadbfd674afbc37ecn;
 ```
 
 In a zkApp, this code can be used to prove that you know a secret value whose hash is publicly known without revealing the secret.
 The code is plain TypeScript and is executed as normal TypeScript. You might call o1js an _embedded domain-specific language (DSL)_.
 
-o1js provides data types and methods that are _provable_: You can prove their execution. 
+o1js provides data types and methods that are _provable_: You can prove their execution.
 
 In the example code, `Poseidon.hash()` and `Field.assertEquals()` are examples of provable methods. Proofs are _zero knowledge_, because they can be verified without learning their inputs and execution trace. Selected parts of the proof can be made public, if it suits your application.
 
 o1js is a general-purpose zk framework that gives you the tools to create zk proofs. It lets you write arbitrary zk programs leveraging a rich set of built-in provable operations, like basic arithmetic, hashing, signatures, boolean operations, comparisons, and more. Use the o1js framework to write zkApps on Mina, smart contracts that execute client-side and have private inputs.
 
-All of the o1js framework is packaged as a single TypeScript library that can be used in major web browsers and Node.js. The best way to get started with o1js is [using the zkApp CLI](./how-to-write-a-zkapp). You can also install o1js from npm with `npm i o1js`. 
+All of the o1js framework is packaged as a single TypeScript library that can be used in major web browsers and Node.js. The best way to get started with o1js is [using the zkApp CLI](/zkapps/writing-a-zkapp/introduction-to-zkapps/how-to-write-a-zkapp). You can also install o1js from npm with `npm i o1js`.
+
+Start your o1js journey by learning about [basic zk programming concepts](/zkapps/o1js/basic-concepts).
+
+## Audits of o1js
+
+* **Third-party audit (Q1/Q2 2024)**. An audit by an external security firm is ongoing and on track to complete in Q2 2024. Results will be shared when available.
+
+* [**Internal audit (Q1 2024)**](https://github.com/o1-labs/o1js/files/15192821/Internal.o1js.audit.Q1.2024.pdf). In March 2024, the o1js team spent roughly two person-weeks to conduct an internal audit of parts of the o1js code base. The audit focused on reviewing core provable code. A number of issues were found and fixed.
+
+Please see our page on [Security and zkApps](/zkapps/writing-a-zkapp/introduction-to-zkapps/secure-zkapps) for more information on ensuring your zkApp is secure.
 
+<<<<<<< HEAD:docs/zkapps/o1js/index.md
```

Comment:
unresolved merge conflict

Reply:
fixed the missed merge conflicts - found 5